test/draftertest.h

```diff
@@ -66,11 +66,11 @@ namespace draftertest {
     public:
         ITFixtureFiles(const std::string& base) : base_(base) {}
 
-        typedef std::auto_ptr<std::istream> stream_type;
+        typedef std::auto_ptr<std::istream> input_stream_type;
```

Comment:
**Note** *changed this name because now that `IIFixtureFiles` can write, it's no longer clear that `stream_type` should be an input stream.*